03-git-github-and-markdown/COURSE_MATERIAL.md

```diff
@@ -319,6 +319,7 @@ _Alternatives are here for general knowledge. No need to learn them._
 - [GitLab](https://gitlab.com/) - My personal favorite
 - [Bitbucket](https://bitbucket.org/) - My least favorite
 - [Gitea](https://gitea.io/) - Very good self-hosted alternative
+- [Forgejo](https://forgejo.org/) - A community owned soft fork of Gitea with upcoming support of *[forge federations](https://forgefed.org/) (see more with the [ForgeFriends project](https://forgefriends.org/))*.
```

Comment:
I think you can remove the italic formatting to keep it consistent with the rest of the list. I will then approve and merge your work, thank you!

Reply:
Should be good now.